docassemble/assemblylinewizard/interview_generator.py

```diff
@@ -1107,44 +1136,42 @@ def indexify(digit):
   else:
     return '[' + str(int(digit)-1) + ']'
 
+# TODO: is this function used or necessary? Looks like no longer referenced
 def should_be_stringified(var_name):
-  has_no_attributes = var_name.find(".") == -1
+  has_no_attributes = not "." in var_name
   is_docket_number = var_name.startswith("docket_numbers[")
   return has_no_attributes and not is_docket_number
 
-def get_person_variables(fieldslist):
+def get_person_variables(fieldslist, people_vars=generator_constants.PEOPLE_VARS, people_suffixes = generator_constants.PEOPLE_SUFFIXES, people_suffixes_map = generator_constants.PEOPLE_SUFFIXES_MAP, reserved_pluralizers_map=generator_constants.RESERVED_PLURALIZERS_MAP, custom_only=False):
   """
   Identify the field names that represent people in the list of
-  DAFields pulled from docx/PDF.
+  string fields pulled from docx/PDF.    
   """
   people = set()
   for field in fieldslist:
-    if is_person(field.variable):
-      people.add(get_person_identifier(field.variable))
-  return people
-
-def is_person(field_name, people_vars=generator_constants.PEOPLE_VARS):
-  """
-  Check if the field name appears to represent a person
-  """
-  # Is it exactly a person variable: e.g., `users`
-  if remove_string_wrapper(field_name) in people_vars:
-    return True
-  if '[' in field_name or '.' in field_name:
-    match_with_brackets_or_attribute = r"(\D\w*)((\[.*)|(\..*))"
-    matches = re.match(match_with_brackets_or_attribute, field_name)
-    if matches:
-      if matches.groups()[0] in people_vars:
-        return True
-      else:
-        endings = [
-          "[0].address_block()",
-          "[0].address.block()",
-          "[0].address.on_one_line()",
-          "[0].",
-        ]
-        if matches.groups()[1] in endings:
-            return True
-
-def get_person_identifier(field_name):
-  pass
+    # fields are tuples for PDF and strings for docx
+    if isinstance(field, tuple):
+      field_to_check = field[0]
```

Comment:
If it's from a PDF, shouldn't it be `field_to_check = map_names(field[0])`, and then remove `map_names` in line 1158?